test/unit/specs/pages/server/collectionbrowserpage.spec.js

```diff
@@ -0,0 +1,15 @@
+
+import CollectionBrowsePage from '@/pages/server/CollectionBrowsePage';
+import render from '../../../test-utils/render';
+
+describe('CollectionBrowsePage', () => {
+  it('should render correct contents', () => {
+    const wrapper = render(CollectionBrowsePage);
+
+    expect(wrapper.find('.browse-page'));
+    expect(wrapper.find({ name: 'header-section' }).vm).toBeDefined();
+    expect(wrapper.find({ name: 'search-grid-filter' }).vm).toBeUndefined();
```

Comment:
Why include this test? The search-grid-filter is never imported to the CollectionBrowserPage component, so this will always pass.